docs/20-batches.md

```diff
@@ -25,8 +25,9 @@ With the config array...
             'crudViewPath' => '@project/modules/crud/views',
             'crudPathPrefix' => '/crud/',
             'crudTidyOutput' => true,
+            'crudActionButtonColumnPosition' => 'right', //left by default
             'crudProviders' => [
-                \schmunk42\giiant\generators\crud\providers\core\OptsProvider::class
+                \schmunk42\giiant\generators\crud\providers\core\OptsProvider::className()
```

Comment:
No need to change this.

Reply:
All generated files use old definitions className() to use with old php versions and I replace it in documentation.